app.py

```diff
@@ -1,12 +1,36 @@
 #!/usr/bin/env python
-from flask import Flask, render_template, Response, request
-
+from flask import Flask, render_template, Response, request, flash
+from flask_sqlalchemy import SQLAlchemy
 import mediapipe_processor
+import psycopg2
 
 Camera = mediapipe_processor.Camera
 
 app = Flask(__name__)
 
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://postgres:8543abcd@localhost/ElecTrap_scoreboard'
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app.secret_key = 'secret string'
+
+db = SQLAlchemy(app)
+
+con = psycopg2.connect(database="ElecTrap_scoreboard", user="postgres", password="8543abcd", host="127.0.0.1", port="5432")
```

Comment:
Config file had better put in another file like `config.ini` and don't commit it due to security.